instance/models/server.py

```diff
@@ -287,8 +289,8 @@ def public_ip(self):
 
         try:
             public_addr = openstack.get_server_public_address(self.os_server)
-        except novaclient.exceptions.NotFound:
-            return None  # This server has been deleted.
+        except Exception:  # pylint: disable=broad-except
```

Comment:
Hey @haikuginger, all exceptions thrown by Requests should inherit from `requests.exceptions.RequestException`. I'm pretty sure we're wrapping everything floating up from urllib3 with that as well.